website/docs/components/link/standalone/partials/accessibility/accessibility.md

```diff
@@ -19,6 +19,4 @@ This section is for reference only. This component intends to conform to the fol
 
 ---
 
-## Support
-
-If any accessibility issues have been found within this component, let us know by [submitting an issue](https://github.com/hashicorp/design-system/issues/new/choose).
\ No newline at end of file
+<Doc::A11ySupport />
```

Comment:
Yay! Is this new?!

Reply:
It was created at some point during the initial website work. I believe this was the only page not using it.